app/screens/PostDetail.tsx

```diff
@@ -3,10 +3,15 @@ import React, {useEffect} from 'react';
 import {Text, View} from 'react-native';
```

Comment:
Small fixes to get this to render and not break. I have not tested the delete button.